src/utils/Card.ts

```diff
@@ -203,7 +236,7 @@ export const cardColorCategory = (card: Card): ColorCategory => {
     return 'Colorless';
   }
   if (colors.length > 1) {
-    return 'Multicolor';
+    return 'Multicolored';
```

Comment:
Found when exploring why some cards didn't show the right value in the Color Category dropdown, looked to be a recent typo. More places used Multicolored including on the Color category type